linux_os/guide/system/software/integrity/fips/enable_dracut_fips_module/ansible/shared.yml

```
@@ -8,6 +8,7 @@
   command: /usr/bin/fips-mode-setup --check
   register: is_fips_enabled
   changed_when: false
+  failed_when: "is_fips_enabled.rc not in [0,1]"
```

This is basically ignoring all errors from `fips-mode-setup`.

How about making this:
```suggestion
  failed_when:  False
```

This way it is more explicit that errors in the command do not cause the task to fail.